internal/browser/browser.go

```diff
@@ -38,6 +38,13 @@ func exists(name string) bool {
 
 //getWindowsChromeLocation Determines different Google Chrome install locations.
 func getWindowsChromeLocation() (string, error) {
+	// if user set chrome path env, check that first
+	chromePath, chromePathFound := os.LookupEnv("CHROME_PATH")
```

Comment:
Lets change this environmental variable to be NVIDIA_CLERK_CHROME_PATH I would rather not have users possibly clutter other env uses.